dfusion_rust_core/src/models/order.rs

```diff
@@ -32,6 +34,27 @@ impl Serializable for u128 {
     }
 }
 
+impl From<Vec<u8>> for Order {
+    fn from (bytes: Vec<u8>) -> Self {
+        let mut bytes = bytes.clone();
+        println!("TODO: Read order bytes: {:?}", bytes);
+        // TODO: Extract order from bytes
+        
+        Order {
```

Comment:
If the format is the same as for the non standing order events, we should try to use the same code for parsing both. But slot_index might be missing here?

Reply:
Yep, I haven't incorporated this PR yet: https://github.com/gnosis/dex-services/pull/207/files

I see on it a `Option<BatchInformation>`. Once it's merged to master, I'll bring those changes to this branch and adapt it so I add a `None` in there.

In that PR I don't see a parsing from the packed form of the events, that is what I implemented in this one (`From<Vec<u8>>`). 

David created the `From<Entity` and `From<Arc<Log>>`.